### PR TITLE
Meta: Enable prettier on CSS and GQL

### PR DESCRIPTION
**Pull request by fregante**
https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +21,10 @@
 		"watch:webpack": "cross-env NODE_OPTIONS=\"--loader tsx --no-warnings=ExperimentalWarning \" webpack --mode=development --watch"
 	},
 	"stylelint": {
-		"extends": "stylelint-config-xo",
+		"extends": [
+			"stylelint-config-xo",
+			"stylelint-config-prettier"
```
Comment (busches) [inline package.json:26]: Sounds like we don't need this if we update our stylelint config in preparation of their next release.

Comment (fregante) [reply package.json:26]: Update what? We're already using the latest versions 

https://github.com/xojs/stylelint-config-xo/issues/19

Comment (busches) [reply package.json:26]: Ah, we're bringing it in via XO, nevermind.

File: source/features/table-input.css [hunk 2]
```diff
@@ -37,15 +37,37 @@
...
+.rgh-tic:is(:nth-of-type(5n + 1)):has(~ :hover:nth-of-type(5n + 1)) div,
+.rgh-tic:is(:nth-of-type(5n + 1), :nth-of-type(5n + 2)):has(
+		~ :hover:nth-of-type(5n + 2)
+	)
+	div,
+.rgh-tic:is(
+		:nth-of-type(5n + 1),
+		:nth-of-type(5n + 2),
+		:nth-of-type(5n + 3)
+	):has(~ :hover:nth-of-type(5n + 3))
```
[the remark this answers is not in the document]
Comment (fregante) [reply source/features/table-input.css:49]: Prettier rules

[the remark this answers is not in the document]
Comment (fregante) [reply source/features/table-input.css:49]: Maybe this is a good file to disable Prettier on

**Pull request merged**